### DISCUSSION TITLE
Triton Meetup at the NVIDIA Headquarters on April 30th 3:00 - 6:30 pm

### DISCUSSION THREAD

**Discussion by meleegy** [triton-inference-server/server#7057] [Announcements]
![ai-for-dev-kv-inference-devnews-1920x1080](https://github.com/triton-inference-server/server/assets/32750279/b4a3ad3f-ba85-4c67-bd02-519fc2ec4cc8)
We conducted the Triton user meetup at the **NVIDIA Headquarters on April 30th 3:00 - 6:30 pm**. You can download the presentation slides [here](https://drive.google.com/file/d/1MkVaDSHOiS3IO4wSNeCYrxvAnih4G5zL/view?usp=drive_link).

**Comment by raybellwaves** [comment]
Sadly I missed this one. Was there any recordings or notes from the meet-up?

**Reply by rmccorm4** [reply]
Hi @raybellwaves, thanks for reaching out!

@meleegy do we have any resources from the meetup to share externally?

**Reply by meleegy** [reply]
Hi @raybellwaves , I just updated the announcement with link to the meetup slides.

**Reply by asamadiya** [reply]
Thanks, @meleegy. If you have the recording to share too, that'd be super useful.
